js/constants/appConfig.js

```diff
@@ -199,6 +199,11 @@ module.exports = {
     'payments.minimum-visits': 1,
     // "Add funds to your wallet" -- Limit to once every n days to reduce nagging.
     'payments.notification-add-funds-timestamp': null,
+    'payments.notification-backup-keys-timestamp': null, // time when to show backup notification reminder
+    'payments.notification-backup-keys-count': 0, // number of times user has been notified to backup wallet
+    // the intervals in days that the user should be reminded to backup wallet
+    // if an out of bounds index is selected, it should default to monthly
+    'payments.notification-backup-keys-interval': [7, 14],
```

Comment:
the `settings` / `defaultSettings` object is supposed to only be used for settings that are exposed in about:preferences such that the user can modify them. 

can we avoid adding more non-user-exposed payments flags/timestamps to settings? unless i'm misunderstanding why they were added here in the first place.

seems like they should go in appState.ledger.about or appState.ledger.info

Reply:
I wondered about that myself. @mrose17 or @NejcZdovc would have to reply on that.

Reply:
yes, let's move this into the state